src/build.sh

```diff
@@ -247,9 +254,21 @@ function full_cleanup()
   cmd_manager "$flag" "$cmd"
 }
 
+function from_sha()
+{
+  local flag="$1"
+  local env_path="$2"
+  local sha="$3"
+  local cmd
+
+  flag=${flag:-'SILENT'}
+  cmd="git rebase ${sha} --exec 'yes \"\" | make prepare${env_path}; make${env_path}'"
+  cmd_manager "$flag" "$cmd"
+}
```

Comment:
> If I am not mistaken, we should find a way to make the --from-sha option only automate the process of the user directly running git rebase -i, stopping in every commit, and running kw build

I'm not sure if I got what you mean here... As a user of this feature, I want to use: `kw b --from-sha <SHA>`, and I only want to stop in the middle of the rebase to take any action if it fails to build for some reason.

> @rodrigosiqueira, do you have any thoughts on this?

I liked the second approach since it will bring all of the kw build benefits, such as the parallel build.